addons/laser/functions/fnc_getLaserCode.sqf

```diff
@@ -0,0 +1,20 @@
+#include "..\script_component.hpp"
+/*
+ * Author: johnb43
+ * Gets the laser code of a laser source.
+ *
+ * Argument:
+ * 0: Laser source <OBJECT>
+ *
+ * Return Value:
+ * Laser code <NUMBER>
+ *
+ * Example:
+ * player call ace_laser_fnc_getLaserCode;
+ *
+ * Public: Yes
+ */
+
+params ["_laserSource"];
```

Comment:
```suggestion
params [["_laserSource", objNull, [objNull]]];

if (_lastSource == objNull) exitWith { -1 };
```

Do something like this?

Reply:
What about if the unit/vehicle is valid, but has no laser active - should it return -1 or the default code? I feel -1 is more appropriate.

Reply:
Does the default code get set when the laser activates? We don't want it returning -1 just because they're on the default code